### PR TITLE
(DOCSP-36734): Add JSON configuration page for Atlas Search Nodes

### PR DESCRIPTION
**Pull request by davidhou17**
- [DOCSP-36734](https://jira.mongodb.org/browse/DOCSP-36734)
- [LATEST BUILD LOG](https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&amp;jobId=65d8f4217111165982b71962)

Staging:
- [JSON landing page](https://preview-mongodbdavidhou17.gatsbyjs.io/atlas-cli/DOCSP-36734/reference/json/)
- [Search Nodes Configuration File](https://preview-mongodbdavidhou17.gatsbyjs.io/atlas-cli/DOCSP-36734/reference/json/search-nodes-config-file/)
### Self-Review Checklist

- [ ] Check that the submodule pulled in the right changes (if applicable).
- [ ] [Define](https://wiki.corp.mongodb.com/display/DE/Taxonomy+tagging+instructions) taxonomy [values](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) at top of page.
- [ ] Add genre facets (tutorial or reference), as in this [example PR](https://github.com/10gen/cloud-docs/pull/5042).
- [ ] Add programmingLanguage (if necessary).
- [ ] Add meta keywords (if necessary).
- [x] Resolve any new warnings or errors in the build.
- [x] Proofread for spelling and grammatical errors.
- [x] Check staging for rendering issues.
- [x] Confirm links are working.

### CODE REVIEW

File: source/reference/json/search-nodes-config-file.txt [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+``.json`` configuration files. 
+
+.. _search-nodes-settings:
+
+Search Nodes Settings
+---------------------
+
+You can specify the following settings in the search nodes
+configuration file. For a full list of settings and 
+descriptions, refer to the :oas-atlas-op:`API specification
```
Comment (JuliaMongo) [inline source/reference/json/search-nodes-config-file.txt:25]: see the

File: source/reference/json/search-nodes-config-file.txt [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can use a configuration file to specify the required 
+settings for creating search nodes on a {+cluster+} 
+by using the {+atlas-cli+}. The {+atlas-cli+} accepts 
+``.json`` configuration files.
```
Comment (JuliaMongo) [inline source/reference/json/search-nodes-config-file.txt:16]: Suggestion:

To create or update search nodes in a {+cluster+} you can
create a ``.json`` configuration file for search nodes and pass this file to a command with {+atlas-cli+}.

File: source/reference/json/search-nodes-config-file.txt [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+   :header-rows: 1 
+   :widths: 20 10 70 
+
+   * - Field 
+     - Type 
+     - Description 
+
+   * - ``specs``
+     - Array of objects
+     - Settings to configure search nodes for your cluster.
```
Comment (JuliaMongo) [inline source/reference/json/search-nodes-config-file.txt:38]: {+cluster+}

File: source/reference/json/search-nodes-config-file.txt [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+
+   * - ``specs.instanceSize``
+     - string
+     - Hardware specification for the search node instance sizes.
```
Comment (JuliaMongo) [inline source/reference/json/search-nodes-config-file.txt:42]: I think this setting warrants adding a link to the search nodes information where NVME is discussed. Do we have such a place in the docs? I would link to it.

File: source/reference/json/search-nodes-config-file.txt [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+       This setting uses the following format:
+
+       - ``<instance-size>_HIGHCPU_NVME``
+       - ``<instance-size>_LOWCPU_NVME``
+
+       For example, ``S20_HIGHCPU_NVME``.
+
+   * - ``specs.nodeCount``
+     - integer
+     - Number of search nodes in the cluster.
```
Comment (JuliaMongo) [inline source/reference/json/search-nodes-config-file.txt:52]: Use a replacement for cluster

**Review by JuliaMongo — APPROVED** [review]
I also suggest that you add a link to the docs about search nodes.

File: source/reference/json/search-nodes-config-file.txt [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+Create Search Nodes
+~~~~~~~~~~~~~~~~~~~
+
+The following example creates search nodes for the {+cluster+} named
+``myCluster`` using a JSON configuration file named 
+``search-nodes-config.json``:
+
+.. code-block::
+
+    atlas clusters search nodes create --clusterName myCluster --file /search-nodes-config.json
```
Comment (jeroenvervaeke) [inline source/reference/json/search-nodes-config-file.txt:83]: if your file is in the same directory it should be either `./search-nodes-config.json` or `search-nodes-config.json`

**Review by jeroenvervaeke — CHANGES_REQUESTED** [review]

**Review by jeroenvervaeke — APPROVED** [review]
LGTM

**Pull request merged**